### ISSUE TITLE
[BUG] raise dead cooldown from dead ghoul

### ISSUE THREAD

**Issue by knasen88** [ClassicWoWCommunity/cata-classic-bugs#402]
#### 1.1. Description
When a ghoul currently dies as unholy, it doesn't incur the 30 second cooldown on the spell and you can instantly resummon it.
  #### 1.2. How to Reproduce
      1. let pet die
      2. observe how you can instantly resummon it
  #### 1.3  Source Material
just let ur pet die on live servers see that you can instantly resummon it.
#### 2.1. Description
30 second CD whenever ghoul dies (or 3 minutes for non unholy i guess)
  #### 2.2 Source Material
https://www.youtube.com/watch?v=zDgk9-N2bMA
https://youtu.be/paskamMBWZ0?t=1089 
https://youtu.be/zDgk9-N2bMA?t=35 
https://youtu.be/zDgk9-N2bMA?t=84 
https://youtu.be/BYK6OkFwbv8?t=54 
https://youtu.be/BYK6OkFwbv8?t=235
https://youtu.be/BYK6OkFwbv8?t=334
https://youtu.be/BYK6OkFwbv8?t=503
https://youtu.be/BYK6OkFwbv8?t=581
https://youtu.be/ufVSmB_hH0s?t=171
https://youtu.be/ufVSmB_hH0s?t=318
https://youtu.be/ufVSmB_hH0s?t=376
https://youtu.be/ufVSmB_hH0s?t=510
https://youtu.be/_YMPcyq4iVQ?t=279
https://youtu.be/_YMPcyq4iVQ?t=367
https://youtu.be/haY0S9aaoh0?t=283
https://youtu.be/0roGbL3ljXw?t=56
https://web.archive.org/web/20140331125324/http://forums.elitistjerks.com/topic/71616-unholy-dps-335-king-of-anything
_This talent should only be taken if you have trouble keeping your Ghoul alive. If you do, it becomes invaluable, as a dead Ghoul is a large portion of your dps down the drain (due to 30 seconds of absolutely zero Ghoul damage,_

**Comment by Cleavis** [comment]
more source material for reference
https://youtu.be/ZHL0BaW3Ye8?t=504
https://youtu.be/DJAYULMSAzw?t=537
https://youtu.be/paskamMBWZ0?t=1944

unholy should incur a 30s CD from the time ghoul dies, not from when ghoul is summoned.

**Comment by NuptupTDOW** [comment]
Just throwing this out there, while I can definitely see this as a bug, it sure has been fun doing AoE pulls in dungeons and doing a hard-cast corpse explosion, then re-summoning and doing another hard-cast corpse explosion back to back (after giving the ghoul enough time to actually spawn and not dropping diseases). It's some hilariously brutal AoE burst damage when stacked with Death and Decay and your typical Unholy Disease damage.

**Comment by JamminL** [comment]
Added some sources

**Comment by ollipopped** [comment]
pls fix this, impacts player power in arena, raid, dungeons...what else do you need?

**Comment by Zugginlol** [comment]
this is not fixed @CharWS

**Comment by Zugginlol** [comment]
@JamminL

**Comment by Teebown** [comment]
Comments on blight club suggest it's fixed
![Uploading Screenshot_20221111_125949.jpg…]()

**Comment by Zugginlol** [comment]
Nah, only a successful ghoul corpse explode puts ghoul on 30s cd. Any other means of death do not put the ghoul on cd. 

As a sidenote, ghoul abilities no longer reset cd IN OPEN WORLD / IN BATTLEGROUNDS. Random change. Does not affect arenas

**Comment by Zugginlol** [comment]
ghoul cd's are not supposed to reset on cooldown on resummon.

Gargoyle also doesn't come straight down to the ground level on summon, as well as death grip pulling players in front of wherever the DK is facing instead of melee range.

**Comment by dummyaccounter** [comment]
> ghoul cd's are not supposed to reset on resummon.

Can you post evidence of this?

**Comment by KhalGH** [comment]
> ghoul cd's are not supposed to reset on resummon.

Ghoul's CD do supposed to reset on resummon.
[https://www.youtube.com/watch?v=ufVSmB_hH0s](https://www.youtube.com/watch?v=ufVSmB_hH0s)
Dk use Gnaw+CE at 2:50, ghoul gets resummoned at 3:33 with Gnaw w/o CD
Similar to [https://www.warcraftmovies.com/movieview.php?id=160059](https://www.warcraftmovies.com/movieview.php?id=160059) (3:11-3:50)
Not sure if it's reported but it's bugged in current build.

**Comment by JamminL** [comment]
> > ghoul cd's are not supposed to reset on resummon.
> 
> Ghoul's CD do supposed to reset on resummon. https://www.youtube.com/watch?v=ufVSmB_hH0s Dk use Gnaw+CE at 2:50, ghoul gets resummoned at 3:33 with Gnaw w/o CD Similar to https://www.warcraftmovies.com/movieview.php?id=160059 (3:11-3:50) Not sure if it's reported but it's bugged in current build.

Pet abilities are resetting only in arenas, as they did back then

**Comment by knasen88** [comment]
still bugged, doesn't go on CD. Allegedly it does if you corspe explode it so they "partially" fixed it, but main issue is still there.

**Comment by knasen88** [comment]
patch 3.1 @CharWS 
> * Raise Dead: Duration lowered to 60 seconds and cooldown lowered to 3
    minutes. The cooldown on this ability now begins when the pet dies
    rather than when it is summoned.

**Comment by JamminL** [comment]
Archiving old non-relevant issues in preparation for MOP. If I made a mistake please comment on it.
